serverless/README.md

```diff
@@ -69,20 +71,22 @@ To further configure your plugin, use the following custom parameters:
 | `enableDDTracing`       | Enable tracing on Lambda function using dd-trace, Datadog's APM library. Defaults to `true`. The Datadog Forwarder Lambda function must subscribe the function log group.                                                                                                                                                                                                                                                                                                                           |
 | `enableDDLogs`          | Enable Datadog log collection for the Lambda function. Note: This setting has no effect on logs sent via the Datadog Forwarder. Defaults to true.                                                                                                                                                                                                                                                                                                                                                   |
-| `service`               | When set, the macro adds a `service` tag to all Lambda functions with the provided value.                                                                                                                                                                                                                                                                                                                                                                                                           |
-| `env`                   | When set, the macro adds an `env` tag to all Lambda functions with the provided value.                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `service`               | When set along with a `forwarderArn`, the macro adds an `service` tag to all Lambda functions with the provided value. When set along with the `extensionLayerVersion`, the macro adds a `DD_SERVICE` environment variable is added to all lambda functions with the provided value.                                                                                                                                                                                                                |
```

Comment:
```suggestion
| `service`               | When set along with the `extensionLayerVersion`, the macro adds a `DD_SERVICE` environment variable to all lambda functions with the provided value. When set along with the `forwarderArn`, the macro adds a `service` tag to all Lambda functions with the provided value.                                                                                                                                                                                                                |
```

Reply:
nit: I thought maybe the extension sentence should come first and I also suggested some rewording.